### PR TITLE
Create list of raindrops retreived from API

### PR DESCRIPTION
**Pull request by julianastahelin**
Raindrop API URL: https://developer.raindrop.io/

### CODE REVIEW

**Review by artu-hnrq — COMMENTED** [review]
Keep up the great work!

File: components/core/references/card.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Raindrop } from "./panel"
```
Comment (artu-hnrq) [inline components/core/references/card.tsx]: What is a Raindrop in panel?

File: components/core/references/panel.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
+'use client'
+import { useState } from "react"
+import { Button } from "@/components/ui/button"
+import { ChevronLeft, ChevronRight } from "lucide-react"
```
Comment (artu-hnrq) [inline components/core/references/panel.tsx]: Organize all icons to be imported from sabe module

File: components/core/references/panel.tsx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    title: string
+    link: string
+    created: string
+    tags: string[]
+    type: string
+    cover: string
+    domain: string
+}
+
+export interface Raindrops {
```
Comment (artu-hnrq) [inline components/core/references/panel.tsx]: Consider rename it to `CollectionRaindropsResponse` and move to Types

File: app/globals.css [hunk 4]
```diff
@@ -73,4 +73,8 @@
   body {
     @apply bg-background text-foreground;
   }
-}
\ No newline at end of file
+}
+
+.small-caps {
```
Comment (artu-hnrq) [inline app/globals.css]: Consider use [tailwind arbitrary proprieties][1] in a component instead a css class

  [1]: https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties

File: components/core/skills/badge-list.tsx [hunk 5]
```diff
@@ -1,5 +1,6 @@
-import { Skill } from "@/types/core"
-import { SkillBadge, ToggleSkill, SkillIsActive } from '@/components/skills'
+import { SkillBadge, ToggleSkill, SkillIsActive } from '@/components/core/skills'
```
Comment (artu-hnrq) [inline components/core/skills/badge-list.tsx]: Let's name boolean-return functions as questions, though `IsActiveSkill` seems better, I'd say

File: components/site/icons/favicon.tsx [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export function Favicon({domain}: {domain: string}) {
```
Comment (artu-hnrq) [inline components/site/icons/favicon.tsx]: It can be slightly better

**Review by artu-hnrq — CHANGES_REQUESTED** [review]
We are almost there! Keep going!

File: components/core/references/panel.tsx (hunk 2 above)
Comment (julianastahelin) [reply components/core/references/panel.tsx]: ok! done

File: app/globals.css (hunk 4 above)
Comment (julianastahelin) [reply app/globals.css]: Good idea. Done

File: components/site/icons/favicon.tsx (hunk 6 above)
Comment (julianastahelin) [reply components/site/icons/favicon.tsx]: Renamed it to ExternalDomainFavicon and refactored.

**Review by artu-hnrq — APPROVED** [review]

**Pull request merged**